src/configuration-store/hybrid.store.ts

```diff
@@ -0,0 +1,59 @@
+import { logger } from '../application-logger';
+
+import { IAsyncStore, IConfigurationStore, ISyncStore } from './configuration-store';
+
+export class HybridConfigurationStore<T> implements IConfigurationStore<T> {
+  servingStore: ISyncStore<T>;
+  persistentStore: IAsyncStore<T> | null;
+
+  constructor(servingStore: ISyncStore<T>, persistentStore: IAsyncStore<T> | null) {
+    this.servingStore = servingStore;
+    this.persistentStore = persistentStore;
+  }
+
+  /**
+   * Initialize the configuration store by loading the entries from the persistent store into the serving store.
+   */
+  async init(): Promise<void> {
+    if (this.persistentStore) {
+      if (!this.persistentStore.isInitialized()) {
+        /**
+         * The initial remote request to the remote API failed
+         * or never happened because we are in the cool down period.
+         *
+         * Shows a log message that the assignments served from the serving store
+         * may be stale.
+         */
+        logger.warn(
+          'Persistent store is not initialized from remote configuration. Serving assignments that may be stale.',
+        );
+      }
+
+      const entries = await this.persistentStore.getEntries();
+      this.servingStore.setEntries(entries);
+    }
+  }
+
+  public isInitialized(): boolean {
+    return this.servingStore.isInitialized() && (this.persistentStore?.isInitialized() ?? true);
+  }
+
+  public get(key: string): T {
+    if (!this.servingStore.isInitialized()) {
+      logger.warn('getting a value from a ServingStore that is not initialized.');
+    }
+    return this.servingStore.get(key);
+  }
+
+  public getKeys(): string[] {
+    return this.servingStore.getKeys();
+  }
+
+  public async setEntries(entries: Record<string, any>): Promise<void> {
```

Comment:
would `Record<String, unknown>` make more sense? I never know between those two with TypeScript 😞 

AI says:
> In summary, use `unknown` when you want to represent a value of `any` type while maintaining type safety, and use any only as a last resort when you have no other choice.

![image](https://github.com/Eppo-exp/js-client-sdk-common/assets/417605/089d1484-eda9-4688-8eaa-cbd77d7321f5)

Reply:
Can't we use `T` here instead of `any`?